www/index.php

```diff
@@ -14,6 +14,13 @@
 	<meta name="msapplication-TileColor" content="#da532c">
 	<meta name="msapplication-config" content="/assets/images/icons/browserconfig.xml">
 	<meta name="theme-color" content="#ffffff">
+	<!-- Meta tags for previewing in Discord, or other such apps. -->
+	<meta property="og:title" content="Satisfactory Tools" />
+	<meta property="og:type" content="website" />
+	<meta property="og:url" content="https://www.satisfactorytools.com/" />
+	<meta property="og:image" content="https://www.satisfactorytools.com/assets/images/icons/android-chrome-512x512.png" />
+	<meta property="og:description" content="A collection of powerful tools for planning and building the perfect base. Calculate your production or consumption, browse items, buildings, and schematics and share your builds with others!" />
```

Comment:
I'd rather go for some MetaService with such params, and update those when needed, like when you're browsing an building (codex) there should be it's image, description and title might look like:

```Satisfactory Tools | Codex | BuildingName```

Just an proposal, but nice to have imo

Reply:
would that work without server side rendering tho? We don't have that yet. We'll probably implement some PHP or SSR to do this later.

Reply:
I'll look into it, this was a temporary idea until server side rendering is implemented, but are you suggesting that server side rendering is not necessary?

Reply:
As far as i know, the "modern" parsers are fine with.

Reply:
For now let's do the simple solution and update that later. Better some meta than no meta :smile: